### PR TITLE
Docs for dashboard

### PR DESCRIPTION
**Pull request by rachaelbrownie**
resolves [BOP-723]

[BOP-723]: https://mirantis.jira.com/browse/BOP-723?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/docs/reference/ui.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Dashboard add-on for MKE
+The MKE Dashboard add-on provides a UI for kubernetes resource management. 
+
+Future: User management and administrative configurations will also be possible through the dashboard.
+
+[INSERT SCREENSHOT]
```
Comment (rachaelbrownie) [inline content/docs/reference/ui.md:6]: Not sure if we want this? 👍 or 👎 ?

File: content/docs/reference/ui.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+- Basic kubernetes management of the following (View, Create, Update + Delete)
+ - Configurations: ConfigMaps
+ - Controllers: ReplicaSets, ReplicationControllers, StatefulSets, Jobs, CronJobs, Daemonsets + Deployments
+ - Namespaces
+ - Nodes
+ - Pods
+ - Services
+ - ServiceAccounts
+ - Storage: StorageClasses, PersistantVolumes + PersistantVolumeClaims
+- View kubernetes resources by namespace, or all namespaces at once
```
Comment (rachaelbrownie) [inline content/docs/reference/ui.md:29]:
```suggestion
 - View kubernetes resources by namespace, or all namespaces at once
```

File: content/docs/reference/ui.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# Dashboard add-on for MKE
```
Comment (ry4nz) [inline content/docs/reference/ui.md:1]: Let's specify that this addon is not available until alpha.2.0

File: content/docs/reference/ui.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+## Installation
+
+To install the Dashboard add-on do this
+```
+[TBD]
+
+```
+
+## 4.0.0-apha.1v
```
Comment (quadespresso) [inline content/docs/reference/ui.md:16]: Typo?

Not sure what was intended here, eg,

```
## 4.0.0-alpha.1v
```

or

```
## v4.0.0-alpha.1
```

or something else

**Review by quadespresso — CHANGES_REQUESTED** [review]

File: content/docs/reference/ui.md (hunk 4 above)
Comment (rachaelbrownie) [reply content/docs/reference/ui.md:16]: Glad you made a comment, it should actually be alpha2. how about ?

```suggestion
## 4.0.0-alpha.2 release
```

File: content/docs/reference/ui.md (hunk 3 above)
Comment (rachaelbrownie) [reply content/docs/reference/ui.md:1]: How about

```suggestion
# Dashboard add-on for MKE [`v4.0.0-alpha.2.0`+]
```
Or we can make a separate folder or something for it ? 🤷

File: content/docs/reference/ui.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+[INSERT SCREENSHOT]
+
+## Installation
+
+To install the Dashboard add-on do this
+```
+[TBD]
+
+```
```
Comment (rachaelbrownie) [inline content/docs/reference/ui.md:14]: who can answer this?

File: content/docs/reference/ui.md (hunk 1 above)
Comment (brianbancroft) [reply content/docs/reference/ui.md:6]: Yes.

File: content/docs/reference/ui.md (hunk 2 above)
Comment (brianbancroft) [inline content/docs/reference/ui.md:29]: Whitespace in this file seems uneven

Comment (quadespresso) [reply content/docs/reference/ui.md:29]: I'm strongly inclined to say that all the lines with a single leading space should have a double leading space. It makes sense (to me, at this particular moment in time) that lines 21-28 would all be bullets subordinate to line 20. And that line 29 would be a fresh "level 1" bullet.

I defer to the original author :slightly_smiling_face:

File: content/docs/reference/ui.md (hunk 5 above)
Comment (brianbancroft) [reply content/docs/reference/ui.md:14]: Leave as-is. Out of scope for this PR. Will be solved as part of [BOP-740](https://mirantis.jira.com/browse/BOP-740)

Comment (rachaelbrownie) [reply content/docs/reference/ui.md:14]: that's a good point, we could add later

**Comment by ry4nz** [comment]
Could you add a link in https://github.com/Mirantis/mke-docs/blob/main/content/releases/README.md with the page you created for discoverability ?

**Comment by rachaelbrownie** [comment]
Ryan has also requested that i add a "what's new" section for things that have a major change compared to MKE4. So I will add that in the next day.

File: content/docs/reference/ui.md (hunk 1 above)
Comment (rachaelbrownie) [reply content/docs/reference/ui.md:7]: I will add this as soon as we have the icons added to the MDS.

**Review by ry4nz — APPROVED** [review]

**Review by quadespresso — APPROVED** [review]

**Pull request merged**